framework/helpers/BaseHtml.php

```diff
@@ -660,12 +660,20 @@ public static function fileInput($name, $value = null, $options = [])
      * the attributes of the resulting tag. The values will be HTML-encoded using [[encode()]].
      * If a value is null, the corresponding attribute will not be rendered.
      * See [[renderTagAttributes()]] for details on how attributes are being rendered.
+     * The following special options are recognized:
+     *
+     * - `doubleEncode`: whether to encode HTML entities in `$value`. If `false`, HTML entities in `$value` will not
```

Comment:
of course. You are using the word 'double' so people are looking for that word in the definition. Why is it `doubleEncode` and not just named `encode`